packages/host/tests/integration/search-index-test.gts

```diff
@@ -1088,59 +1104,56 @@ module(`Integration | search-index`, function (hooks) {
       },
     });
 
-    let { embeddedHtml, _embeddedHtmlByClassHierarchy } =
+    let { embeddedHtml } =
```

Comment:
With this change `embeddedHtml` that the `getInstance` returns is a JSON, exactly like we save it in the boxel index table. There is no idea of a "default" embedded template so the consumer of `getInstance` should decide which one to use.